CHANGELOG.md

```diff
@@ -1,3 +1,17 @@
+##2015-11-18 Release 1.3.0
```

Comment:
@borcean Can you remove the date? We've come to the conclusion that we will no longer be added the date to the CHANGELOG since it's become a pain to release at the exact date.